recipe/build.sh

```diff
@@ -55,7 +54,5 @@ pushd cctools_build_final
     --disable-static \
     --with-libtapi=${PREFIX} \
     --enable-shared || (cat config.log && cat config.status && false)
-  cat config.log
-  cat config.status
-  make -j${CPU_COUNT} ${VERBOSE_AT} -k
+  make -j${CPU_COUNT} ${VERBOSE_AT}
```

Comment:
For context: https://github.com/h-vetinari/cctools-and-ld64-feedstock/commit/28196b162f033296d154d8c666c39c6427707b14 was explicitly for debugging purposes; it spams the logs quite extensively (I would have thought it's good enough to show them in the error case).

And `-k` makes it much harder to chase down the actual error, because the build keeps going until it can't (forcing to scroll through tons of unrelated logs before finding actual errors).

I would have considered both these things an improvement, but if you want...

Reply:
Yes, they were for debugging purposes, but keeping them there is not much of a burden and helps debug the next error easily.

Reply:
I can see the marginally positive utility of the `cat`s, but for `-k`, it's IMO really more of a burden than a help. In any case, I did as you asked.

Reply:
Yeah, I don't see any usefulness of `-k`. Sorry about selecting all three lines